<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="IN2070121W2" name="com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest" tests="4" failures="1" timestamp="23 Jul 2018 09:29:14 GMT" time="131.075" errors="0">
  <testcase name="getUserScreen" time="36.489" classname="com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest"/>
  <testcase name="getGoalScreen" time="1.693" classname="com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest"/>
  <testcase name="verifyPageLevelErrorMsg" time="5.524" classname="com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest"/>
  <testcase name="verifyFieldLevelErrorMsg" time="87.369" classname="com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest">
    <failure type="java.lang.AssertionError" message="expected [Please enter a valid amount  0-999,999,999,999] but found [Please enter a valid amount 0-999,999,999,999]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a valid amount  0-999,999,999,999] but found [Please enter a valid amount 0-999,999,999,999]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest.verifyFieldLevelErrorMsg(LargePurchaseGoalActivityQuestionnairePageTest.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFieldLevelErrorMsg -->
</testsuite> <!-- com.ey.test3po.testcases.negativepagelevelcases.LargePurchaseGoalActivityQuestionnairePageTest -->
